docs/_source/docs/templates.rst

```diff
@@ -31,6 +31,119 @@ Template. Sceptre User Data is accessible within Templates as
 ``sceptre_user_data`` accesses the ``sceptre_user_data`` key in the Stack
 Config file.
 
+
+Example
+~~~~~~~
+
+While rendering templates with Jinja2, some characters aren't supported by AWS. A workaround for this is to use Jinja to replace `-`, `.` and `_` with other characters to ensure that we produce a valid template. With this we had some plasticity to the template and it becomes easier to add/remove entries on a particular stack.
+
+With this templating some problems will arise, such empty strings as parameters. In the following example you can find a work around for this issues.
+
+This represents a stack to deploy route53 records, it's only showing CNAME and ALIAS records to not get too large.
+
+Stack:
+
+.. code-block:: yaml
+
+    template_path: dns-extras.j2
+    dependencies:
+    - prod/route53/domain-zone.yaml
+    parameters:
+      DomainName: "example.com"!stack_output prod/route53/example-com-zone.yaml::FullDomainName
```

Comment:
there's a dependency to `prod/route53/domain-zone.yaml` but it is not provided as part of this example?  should we just remove it?  if this is an example of creating a jinja2 template then i would suggest reducing the scope.

Reply:
this contribution is a working stack :) I agree with you. I shall replace the dependency with the values to make it easier to the reader